js/piwik.js

```diff
@@ -2783,6 +2785,32 @@ if (typeof window.Matomo !== 'object') {
                 return false;
             }
 
+            /*
+             * Checks if the special query parameter was included in the current URL indicating this
+             * is supposed to be a tracking code install test.
+             */
+            function wasJsTrackingCodeInstallCheckParamProvided()
+            {
+                if (trackerInstallCheckNonce && trackerInstallCheckNonce.length > 0) {
+                    return true;
+                }
+
+                trackerInstallCheckNonce = getUrlParameter(windowAlias.location.href, 'tracker_install_check');
```

Comment:
Do we perhaps want to name this more on brand since it will get logged in server logs and possibly in other tracking tools like GA, so it's clearer/cleaner what the query parameter relates to?

```suggestion
                trackerInstallCheckNonce = getUrlParameter(windowAlias.location.href, 'matomo_tracker_install_check');
```

Reply:
Discussed in person and if needed we can always do this later.